### PR TITLE
Add platform run later

### PR DESCRIPTION
**Pull request by 1-alex98**

### CODE REVIEW

**Review by Geosearchef — APPROVED** [review]

File: src/main/java/com/faforever/client/rankedmatch/Ladder1v1Controller.java [hunk 1]
```diff
@@ -261,7 +259,7 @@ public void onFactionButtonClicked() {
   }
 
   private void setCurrentPlayer(Player player) {
-    playerRatingListener = ratingObservable -> updateRating(player);
+    playerRatingListener = ratingObservable -> Platform.runLater(() -> updateRating(player));
```
Comment (Katharsas) [inline src/main/java/com/faforever/client/rankedmatch/Ladder1v1Controller.java:262]: There are only two calls to `setCurrentPlayer()`, one of which already wraps the entire call in Platform.runLater (remove i guess?).

Comment (Katharsas) [reply src/main/java/com/faforever/client/rankedmatch/Ladder1v1Controller.java:262]: Or i guess you could remove it here and add it there,

**Review by Brutus5000 — APPROVED** [review]

File: src/main/java/com/faforever/client/rankedmatch/Ladder1v1Controller.java (hunk 1 above)
Comment (1-alex98) [reply src/main/java/com/faforever/client/rankedmatch/Ladder1v1Controller.java:262]: Well but this is a listener so not on the same thread than setCurentPlayer

Comment (Katharsas) [reply src/main/java/com/faforever/client/rankedmatch/Ladder1v1Controller.java:262]: Oh right.

**Pull request merged**